src/ex6/server/server/db/config/config.json

```diff
@@ -2,7 +2,7 @@
   "development": {
     "username": "root",
     "password": "password",
-    "database": "todo_db",
+    "database": "exercise6",
```

Comment:
this actually crashed the project the first time I loaded it - we use todo_db as a standard for all exercises
if you need to reset the DB you can just remove the image and recreate it
`docker run -p 3306:3306 --name tododb -e MYSQL_ROOT_PASSWORD=password -e MYSQL_ROOT_HOST=% -e MYSQL_DATABASE=todo_db -d mysql/mysql-server`